### ISSUE TITLE
[CLIENT] [NX/VITA] R_AliasSetupFrame dprint spam

### ISSUE THREAD

**Issue by tyleryoung88** [nzp-team/nzportable#865]
When developer mode is set to 1 or above, there are some console spams regarding the frames of specific assets. The two that I saw during my play testing were - 

`"R_AliasSetupFrame: no such frame 9 for 'models/props/rebar.mdl'"`
and
`"R_AliasSetupFrame: no such frame 2 for 'models/props/teddy.mdl"`

**To Reproduce**
1. type "developer 1" in the console
2. observe the console output

**Expected behavior**
There should not be any warnings from these assets

**Affected Platforms**
This print seems to only be present on Quakespasm port. 
- PS Vita (1k, 2k)
- Nintendo Switch

**Comment by MotoLegacy** [comment]
As stated in the PR comment, please make these separate issues and tag them accordingly. To repeat:

1. Rebar is specific to NDU, please report it as an NDU bug

2. Is a general asset issue because the teddybear sitting frame was accidentally removed

Thanks!

**Comment by blubs** [comment]
Related but a quality of life fix might be to start splitting different categories of `DPrint`s into different `developer` cvar values. 

For example,  when debugging pathfinding, I found unrelated `DPrint`s overwhelming so I broke out the pathfinding `DPrint`s (which are themselves overwhelming) to the `developer`==3 case.

This `R_AliasSetupFrame` is particularly spammy, so might be useful to move into a new graphics-focused `DPrint` category to facilitate focused debugging of features.
